src/Storage/ClaimController.ts

```diff
@@ -53,51 +54,142 @@ export class ClaimController {
     const logger = this.logger.child({ method: 'download' })
 
     logger.trace({ ipfsHashes }, 'Downloading Claims')
-
-    await this.collection.insertMany(ipfsHashes.map(ipfsHash => ({ ipfsHash, claimId: null })), { ordered: false })
+    await this.collection.insertMany(
+      ipfsHashes.map((ipfsHash): Entry => ({
```

Comment:
`ipfsHashes.map((ipfsHash): Entry` => ipfsHash is not of type `Entry` here, it's `string`. The type is inferred though, so you can just avoid declaring the type.